vignettes/adding-support-for-reporting.Rmd

```diff
@@ -35,7 +35,7 @@ my_module <- function(label = "example teal module") {
   module(
     label = label,
     server = function(id, data) {
-      checkmate::assert_class(data, "reactive")
+      assert_reactive(data)
```

Comment:
Nobody is required to assert on `data`, we guarantee that this will be a `reactive`
```suggestion
```